README.md

```diff
@@ -189,17 +209,20 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 | <a href="https://drawkit.com/product/people-working-collaborating-illustrations"><img width='700' src='https://user-images.githubusercontent.com/74038190/184807822-86820ac2-f3f3-4eb2-bf70-3d478f7f1a98.png' alt="People_Working_free_illustrations" /></a> | [People Working](https://drawkit.com/product/people-working-collaborating-illustrations)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• InVision Studio<br>• SVG<br><br> ✌️Author <br> DrawKit |
 | <a href="https://growwwkit.com/illustrations/phonies/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184864644-60ec8f7d-30d3-4f5e-903a-0c8a92fec3f2.png' alt="Phonies_free_illustrations" /></a> | [Phonies](https://growwwkit.com/illustrations/phonies/)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• SVG<br>• AI<br><br> ✌️Author <br> Growww Kit                                            |
 | <a href="https://www.kapustin.co/product/porto"><img width='700' src='https://user-images.githubusercontent.com/74038190/184643045-1b87a006-5c56-4959-a0e2-085aebd4b211.png' alt="Porto_free_illustrations" /></a> | [Porto](https://www.kapustin.co/product/porto)  <br><br> ✔ Free <br><br>Format <br> • AI<br>• PNG<br><br> ✌️Author <br> Craftwork                                                                                             |
-| <a href="https://www.getillustrations.com/illustration-pack/product-hunt-illustrations-free-cat-stickers"><img width='700' src='https://user-images.githubusercontent.com/74038190/185657863-a9b730e5-f2cd-41eb-933a-9693d187b761.png' alt="Product_Hunt_free_illustrations" /></a> | [Product Hunt](https://www.getillustrations.com/illustration-pack/product-hunt-illustrations-free-cat-stickers)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PNG<br>• SVG<br><br> ✌️Author <br> Getillustrations                                                       |
+| <a href="https://www.getillustrations.com/illustration-pack/product-hunt-illustrations-free-cat-stickers"><img width='700' src='https://user-images.githubusercontent.com/74038190/185657863-a9b730e5-f2cd-41eb-933a-9693d187b761.png' alt="Product_Hunt_free_illustrations" /></a> | [Product Hunt](https://www.getillustrations.com/illustration-pack/product-hunt-illustrations-free-cat-stickers)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PNG<br>• SVG<br><br> ✌️Author <br> Getillustrations    
+| <a href="https://drawkit.com/product/product-project-managers-illustrations"><img width='700' src='https://user-images.githubusercontent.com/34191321/194714803-5fb5ea2d-c326-4fc5-85c0-c6220b82ec50.png' alt="product_&_project_managers_free_illustrations" /></a> | [Product & Project Managers](https://drawkit.com/product/product-project-managers-illustrations)  <br><br> ✔ Free <br><br>Format <br> • Figma<br> • SVG<br> • PNG<br><br> ✌️Author <br> DrawKit |
```

Comment:
- change src link to `https://user-images.githubusercontent.com/74038190/194872674-c12a20e6-d262-4a0f-b376-e8eb5643435d.png` in Product & Project Managers

I don't know why there is so much negative space in your screenshot on the left & right.